src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -135,13 +153,20 @@ namespace Microsoft.FSharp.Collections
         ///
         /// <returns>The result sequence.</returns>
         ///
-        /// <example>
+        /// <example id="cache-1">
         /// <code lang="fsharp">
-        ///     let fibSeq = Seq.unfold (fun (a,b) -> Some(a + b, (b, a + b))) (0,1)
-        ///     let fibSeq3 = fibSeq |> Seq.take 3 |> Seq.cache
-        ///     fibSeq3 // evaluates to seq [1; 2; 3], and it will not do the calculation again when called.
-        ///     null |> Seq.cache // trhows exception!
+        /// let fibSeq =
+        ///   Seq.unfold
```

Comment:
I personally would pipe into `unfold`. I'll make that change

Reply:
Note taken, will do in next PR

Reply:
Oh, you did, nevermind.